js/__internal/grids/grid_core/data_controller/module.ts

```diff
@@ -1267,12 +1267,13 @@ export class DataController extends ControllerWithDataMixin {
   }
 
   _createDataSourceAdapter(dataSource) {
-    let remoteOperations = this.option('remoteOperations');
+    let remoteOperations: any = this.option('remoteOperations');
```

Comment:
Why do we add ```any``` type here?
Now current calculated type is very ugly, but maybe the ugly type is better than ```any```?

Reply:
Ok,agree with you
Let's leave the ```any``` type for now